go.mod

```diff
@@ -5,16 +5,19 @@ go 1.16
 require (
 	github.com/Masterminds/squirrel v1.5.3
 	github.com/alexedwards/argon2id v0.0.0-20211130144151-3585854a6387
-	github.com/aws/aws-sdk-go v1.45.5
+	github.com/aliyun/alibaba-cloud-sdk-go v1.62.733 // indirect
```

Comment:
Why update github.com/aliyun/alibaba-cloud-sdk-go?

Reply:
> Why update github.com/aliyun/alibaba-cloud-sdk-go?

To upgrade the sender library, first execute the command `go get -u github.com/casdoor/go-sms-sender`, and then after completing the local development and testing, execute the `go mod tidy` command.

Reply:
@cutedrag0n I have updated the version, see commit: https://github.com/casdoor/casdoor/commit/3456fc6695f70ab48b4e9c3ee1becdb41b9e9dd0

And I also ran `go mod tidy`, there's no that much diff as yours.

Anyway, you don't need to do this again, as I have did this.